src/rules/no-deprecated-classes.js

```diff
@@ -56,12 +56,13 @@ module.exports = {
   },
 
   create (context) {
-    return context.parserServices.defineTemplateBodyVisitor({
+    const sourceCode = context.getSourceCode()
+    return sourceCode.parserServices.defineTemplateBodyVisitor({
       'VAttribute[key.name="class"]' (node) {
         if (!node.value || !node.value.value) return
 
         const classes = node.value.value.split(/\s+/).filter(s => !!s)
-        const source = context.getSourceCode()
+        const source = sourceCode.getText()
```

Comment:
this should be `sourceCode` instance because below `getLocFromIndex` is a method of sourceCode.
https://eslint.org/docs/latest/extend/custom-rules#accessing-the-source-code

Reply:
This is using the `sourceCode` instance. That is exactly what this change is doing. I am not sure what you are suggesting.

Reply:
@zhuiks - it looks like this change request was addressed in b6f4925b1539410fcdcb0ec2f2d2798c218af3d4 

Is there anything else you need before this can be merged?

Reply:
Testing this in a project showed that this line should actually be just 
```suggestion
        const source = sourceCode
```
(or the usage in lines 94-95 adjusted to use `sourceCode` directly

Reply:
Done!
Let's just get this published so people can use it.

Reply:
We should just convert to TypeScript so all of these issues get flagged and we can just go through and fix each one.
If TypeScript is too much we can just go with jsDoc